keyserver/handler.go

```diff
@@ -19,10 +19,22 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 			p.Error(p.ErrorMalformedClientMessage)
 	}
 
+	server.Lock()
+	// check the temporary bindings array first
+	// currently the server allows only one registration/key change per epoch
+	if server.tbs[reg.Username] != nil {
```

Comment:
I agree that the error should be returned to the client to distinguish from the success case. We could return the existing TB along with the auth path, as @c633 suggested.